src/pydocstyle/violations.py

```diff
@@ -252,8 +252,8 @@ def to_rst(cls) -> str:
                                  'mark, or exclamation point', 'not {0!r}')
 D416 = D4xx.create_error('D416', 'Section name should end with a semicolon',
                          '{0!r}, not {1!r}')
-D417 = D4xx.create_error('D417', 'Missing arguments in the docstring',
-                         'argument(s) {0!r} missing in {1!r} docstring')
```

Comment:
Personally I thought the more concise original wording was fine?

Reply:
However, looking at the tests changed, I would stop using ``r`` and use ``s`` format as the quotes round multiple argument names looks silly.

Reply:
Hmm, I wanted to make it clear that it's not just checking for the presence of the arguments, but a valid description too. Otherwise people might comment a parameter and it's type and wonder why pydocstyle is throwing an error.

Reply:
Fair point, that was ambiguous

Reply:
The single quotes around the entire list is a bit weird. Either each parameter should be quotes separately or no quotes at all.

This is a bit weird: 

    argument(s) 'test, x, y' are missing...

These are acceptable: 

    argument(s) 'test', 'x', 'y' are missing...
    argument(s) test, x, y are missing...